src/components/Navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import InputBase from '@material-ui/core/InputBase';
+import SearchIcon from '@material-ui/icons/Search';
+import MenuIcon from '@material-ui/icons/Menu';
+import { useStyles } from './styles';
+import { Link, useHistory } from 'react-router-dom';
+
+export default function Navbar(props){
+    const classes = useStyles();
```

Comment:
you could destructure the properties and use them directly at the JSX code
```suggestion
    const { root, menuButton, navbarTitle, ... etc } = useStyles();
```